src/components/Theme/ScreenLayout.tsx

```diff
@@ -0,0 +1,18 @@
+import { h, Fragment, ComponentType, ComponentChildren, VNode } from 'preact'
+import theme from './style.scss'
+
+type ScreenLayoutProps = {
+  children: ComponentChildren
+  actions?: ComponentType | VNode
+}
+
+const ScreenLayout = ({ children, actions }: ScreenLayoutProps) => {
```

Comment:
Declare the component as `FunctionComponent<Props>` is better:
- You don't need to declare the params type
- The component will have the correct type of `ComponentType<Props>`

(`FunctionComponent` can be imported from `preact` module)

```suggestion
const ScreenLayout: FunctionComponent<ScreenLayoutProps> = ({ children, actions }) => {
```